specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -385,54 +369,39 @@
           "readOnly": true,
           "description": "The properties associated with the identity.",
           "$ref": "#/definitions/IdentityProperties"
-        },
-        "type": {
-          "type": "string",
-          "readOnly": true,
-          "enum": [
-            "Microsoft.ManagedIdentity/userAssignedIdentities"
-          ],
-          "x-ms-enum": {
-            "name": "UserAssignedIdentities",
-            "modelAsString": true
-          },
-          "description": "The type of resource i.e. Microsoft.ManagedIdentity/userAssignedIdentities"
         }
       },
       "required": [
         "location"
       ],
       "allOf": [
         {
-          "$ref": "#/definitions/Resource"
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
```

Comment:
nit: Would TrackedResource be what you are looking for? It is Resource + location + tags. Does have an extra field tags.

Reply:
> These api-version all seems go way back 2 - 5 years. If they are widely used, I would suggest if you want to hold off and apply the correct reference in the next API-version.

It seems like our swagger spec had minor issues and did not reflect the exact behavior of our API so this PR and the previous were addressing these issues. The referencefixes are there to fix the .NET SDK generation.

Reply:
I'll double check that on our side.